src/core/LadspaManager.cpp

```diff
@@ -436,8 +436,8 @@ float LadspaManager::getDefaultSetting( const ladspa_key_t & _plugin,
 				if( LADSPA_IS_HINT_LOGARITHMIC
 							( hintDescriptor ) )
 				{
-					return( exp( log( portRangeHint->LowerBound ) * 0.75 +
-								 log( portRangeHint->UpperBound ) * 0.25 ) );
+					return std::exp(std::log(portRangeHint->LowerBound)
```

Comment:
What? Isn't exp(log(n)) same as n?

Reply:
It's true that `exp(log(n)) == n`; however, the expression is `exp(¾log(a) + ¼log(b))`. It's split across two lines since it's > 120ch with tabs, but it does look rather silly when viewing only the first line, lol.

Full expression:
```cpp
return std::exp(std::log(portRangeHint->LowerBound) * 0.75 + std::log(portRangeHint->UpperBound) * 0.25);
```

Reply:
Mb